src/SIL.LCModel/DomainImpl/OverridesLangProj.cs

```diff
@@ -907,6 +907,31 @@ private void InitializePronunciationWritingSystems()
 					});
 		}
 
+		/// <summary>
+		/// Confirms there is at least one valid writing system in the list of pronunciation writing systems. Removes any writing systems
+		/// in the list that aren't found in this project until a valid one is found.
+		/// </summary>
+		/// <returns>True if a valid writing system is found, otherwise false.</returns>
+		private bool CheckValidPronunciationWritingSystem()
+		{
+			CoreWritingSystemDefinition ws = null;
+			while (m_currentPronunciationWritingSystems.Count > 0)
+			{
+				try
+				{
+					ws = m_currentPronunciationWritingSystems[0];
+					return true;
```

Comment:
So what if there is one good ws, followed by one bad ws?

Reply:
Good catch! I am finding that The dictionary uses the list of CurrentPronunciationWritingSystems, which accesses this member. And if the ws is invalid, I get another crash. I will make the adjustment.